mani_skill/assets/robots/so101/so101.urdf

```diff
@@ -438,7 +438,7 @@
 
   <!-- Joint from base to shoulder -->
   <joint name="shoulder_pan" type="revolute">
-    <origin xyz="0.0388353 -8.97657e-09 0.0624" rpy="3.14159 4.18253e-17 -3.14159"/>
+    <origin xyz="0.0388353 -8.97657e-09 0.0624" rpy="3.14159 4.18253e-17 -3.20159"/>
```

Comment:
e-17 should be 0.

I noticed the official urdfs from the robotstudio used the e-17 values, but we should use 0 (and I'll tell them they should use 0 as well).